include/ServerMainComponent.hpp

```diff
@@ -179,5 +184,10 @@ class ServerMainComponent : public juce::Component
 	bool autostart = false;
 	bool hasStartBeenCalled = false;
 
+	int alarmAckKeyCode = juce::KeyPress::escapeKey;
+	bool alarmAckKeyPressed = false;
+	std::uint32_t alarmAckKeyMaskId = isobus::NULL_OBJECT_ID;
+	std::shared_ptr<isobus::ControlFunction> alarmAckKeyWs;
```

Comment:
Nit (non-blocking): Prefer to order like the above members by size to minimize wasted RAM for memory word alignment